modules/ROOT/pages/server-compatibility-transactions.adoc

```diff
@@ -3,4 +3,44 @@
 :idprefix:
 :idseparator: -
 
-include::partial$block-related-content-sync.adoc[]
+
+include::partial$_std-hdr-sgw.adoc[]
+
+
+// BEGIN -- Page Attributes
+:url-api-method-database-in-batch: http://docs.couchbase.com/mobile/{version-full}/couchbase-lite-swift/Classes/Database.html#/s:18CouchbaseLiteSwift8DatabaseC7inBatch5usingyyyKXE_tKF[Database.inBatch()]
+// END -- Page Attributes
+
+
+// BEGIN -- Page Heading
+:param-topic-group: compatibility
+:param-related: {compatibility--xref}
+include::partial$block-abstract.adoc[]
+// END -- Page Heading
+
+
+== Introduction
+
+Couchbase Server provides the backing data store for Sync Gateway and it is compatible -- see: {compatibility--xref}.
+However, there are considerations you needs to be aware of when using some Couchbase Server features with Sync Gateway.
+One of those instances concerns the use of Couchbase Server Transactions.
+
+
+== Local Transactions
+
+Within the Couchbase Mobile cluster all transactions are local, either to the Couchbase Server or to the Couchbase Lite database:
+
+* Couchbase Server supports server side transactions -- see: {server-transactions--xref}. +
+Changes made on the server within a transaction block are guaranteed to be imported by the Sync Gateway only after all the changes in a transaction are committed.
+
+* Similarly, Couchbase Lite supports transactions that are local to the Couchbase Lite database -- see: {url-api-method-database-in-batch}. +
+Only committed transactions will ever reach Sync Gateway.
+
+Sync Gateway is not transaction-aware.
```

Comment:
You've already stated that transactions won't be imported by Sync Gateway until they are committed, so I don't think it's correct to say that Sync Gateway is not transaction aware.  I think the more important clarification is that mobile replication isn't transaction-aware, or more specifically doesn't replicate transactions atomically.  This isn't just a Sync Gateway consideration - Couchbase Lite doesn't replicate documents transactionally either (to SG, or P2P).

Reply:
@adamcfraser -- changed in line in published version..